packages/compass-components/src/components/toast-body.tsx

```diff
@@ -1,25 +1,26 @@
 import React from 'react';
-import { css, Link, spacing } from '@mongodb-js/compass-components';
+import { Link } from './leafygreen';
+import { css } from '@leafygreen-ui/emotion';
 
-const toastActionStyles = css({
-  textTransform: 'uppercase',
+const toastBodyFlexStyles = css({
+  display: 'flex',
+  flexDirection: 'row',
+});
 
-  // Show the link action always to the right of the text.
-  // We float it so that it does not constantly reposition when
-  // contents, like a count, changes often.
-  float: 'right',
+const toastBodyTextStyles = css({
+  flexGrow: 1,
+});
 
-  // Override LeafyGreen toast button positioning styles.
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  position: 'relative !important' as any,
+const toastActionStyles = css({
+  textTransform: 'uppercase',
+  flexGrow: 0,
+  alignSelf: 'center',
 
   // Remove button styles.
   border: 'none',
   padding: 0,
   margin: 0,
   background: 'none',
-
-  marginLeft: spacing[2],
 });
 
 // TODO(COMPASS-6808): Once we update our `@leafygreen-ui/toast` we can use the
```

Comment:
I guess this is not true and can be removed as the option they added doesn't cover our use cases anyway?

Reply:
You mean the `// Remove button styles.` comment?

I believe it also applies to the border, padding and margin?

Or are you thinking on:

```
// TODO(COMPASS-6808): Once we update our `@leafygreen-ui/toast` we can use the
```

Reply:
The comment I left is for the line with `Once we update our `@leafygreen-ui/toast` we can use the`, yes. This TODO refers to the the `actionElement` property of the leafygreen toast that we are not using here and can't use anyway because it is only allowed for in-progress toast